docs/user_guide/quickstart.rst

```diff
@@ -1,158 +1,135 @@
-flow, and tool parameters based on a target setup module. To understand the
-complete target configuration, see the :ref:`Flows Directory`, :ref:`PDK
-Directory`, and :ref:`Targets Directory` sections of the reference manual and read the source code for
-`asicflow <https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/flows/asicflow.py>`_,
-`freepdk45 <https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/pdks/freepdk45.py>`_, and
-`freepdk45_demo <https://github.com/siliconcompiler/siliconcompiler/blob/main/siliconcompiler/targets/freepdk45_demo.py>`_.
+Skip to `remote run results`_ to see the expected output.
 
-.. note::
+You do not need to configure anything to use the :ref:`remote` flag with these public servers, but you can use the :ref:`sc-configure` command to specify where SiliconCompiler should send remote jobs::
 
-   This example assumes that :ref:`Surelog <surelog>`, :ref:`Yosys <yosys>`, :ref:`OpenROAD <openroad>`, and :ref:`KLayout <klayout>` are all correctly
-   installed. Links to individual tool installation instructions and platform
-   limitations can be found in the :ref:`Tools directory`.
+    sc-configure https://server.siliconcompiler.com
 
-   It also requires downloading and pointing SC to :ref:`FreePDK45 <freepdk45>`, which is bundled
-   with the SiliconCompiler repo. To install, clone the repo and set up an
-   environment variable ``SCPATH`` pointing at the ``siliconcompiler/``
-   directory inside of it:
 
-   .. parsed-literal::
+.. _private:
 
-     git clone -b v\ |release| https://github.com/siliconcompiler/siliconcompiler
-     export SCPATH=$PWD/siliconcompiler/siliconcompiler
+Private Cloud Access
+^^^^^^^^^^^^^^^^^^^^^
 
-   To simplify tool/PDK installation and job scheduling, SiliconCompiler supports a
-   "-remote" option, which directs the compiler to send all steps to a remote
-   server for processing. The "-remote" option relies on a credentials file
-   located at ``~/.sc/credentials`` on Linux or macOS, or
-   at ``C:\\Users\\USERNAME\\.sc\\credentials`` on Windows, which is generated using :ref:`sc-configure`.
+SiliconCompiler also supports private servers which require authentication to access. If you have such a server to connect to, you will need a credentials text file located at `~/.sc/credentials` on Linux or macOS, or at `C:\\Users\\USERNAME\\.sc\\credentials` on Windows. The credentials file is a JSON formatted file containing information about the remote server address, username, and password.
 
-   Remote processing option is enabled by setting the :keypath:`option,remote`
-   parameter to True. ::
+.. code-block:: json
 
-     chip.set('option', 'remote', True)
+   {
+   "address": "your-server",
+   "username": "your-username",
+   "password": "your-key"
+   }
 
-Compilation
-------------
+Use a text editor to create the credentials file. Alternatively you can use :ref:`sc-configure` app to generate it from the command line.
```

Comment:
"You can use a text editor to create the credentials file, or you can run [sc-configure] from the command line and follow the prompts."